dvc/ignore.py

```diff
@@ -399,3 +399,10 @@ def init(path):
         )
 
     return dvcignore
+
+
+def destroy(path):
+    from dvc.utils.fs import remove
+
+    dvcignore = os.path.join(path, DvcIgnore.DVCIGNORE_FILE)
+    remove(dvcignore)
```

Comment:
It makes sense to `init` a dvcignore inside a directory as it does not exist yet, but I am not sure if it makes sense for `destroy`.